modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/example.json

```diff
@@ -6,55 +6,52 @@
       "claimProcessType": "STANDARD_CLAIM_PROCESS",
       "claimDate": "2020-12-18",
       "veteranIdentification": {
-          "mailingAddress": {
+        "currentlyVaEmployee": false,
+        "mailingAddress": {
             "numberAndStreet": "1234 Couch Street",
-            "apartmentOrUnitNumber": "Apt. 22",
+            "apartmentOrUnitNumber": "22",
             "city": "Portland",
             "country": "USA",
-            "zipFirstFive": "12345",
-            "zipLastFour": "6789",
+            "zipFirstFive": "41726",
+            "zipLastFour": "1234",
             "state": "OR"
-          },
-          "currentlyVAEmployee": false,
-          "serviceNumber": "123456789",
-          "emailAddress": {
-              "email": "claimant@example.com",
-              "agreeToEmailRelatedToClaim": true
-          },
-          "veteranNumber": {
-              "telephone": "555-5555",
-              "internationalTelephone": "555-5555"
-          },
-          "vaFileNumber": "12345678"
-      },
-      "changeOfAddress": {
-          "country": "USA",
-          "type": "INTERNATIONAL",
-          "addressLine1": "1234 Couch Street",
-          "dates": {
-            "beginningDate": "2018-11-31",
-            "endingDate": "2019-07-11"
-          },
-          "typeOfAddressChange": "Permanent",
-          "numberAndStreet": "1234 Couch Street",
-          "apartmentOrUnitNumber": "Apt. 22",
-          "city": "Portland",
-          "zipFirstFive": "12345",
-          "zipLastFour": "1234",
-          "state": "OR"
-      },
+        },
+        "serviceNumber": "123456789",
+        "emailAddress": {
+            "email": "valid@somedomain.com",
+            "agreeToEmailRelatedToClaim": true
+        },
+        "veteranNumber": {
+            "telephone": "5555555555",
+            "internationalTelephone": "5555555555"
+        },
+        "vaFileNumber": "AB123CDEF"
+    },
+    "changeOfAddress": {
+      "dates": {
+          "beginningDate": "2012-11-31",
+          "endingDate": "2013-10-11"
+        },
+        "typeOfAddressChange": "TEMPORARY",
```

Comment:
Looking at the Ticket, does"typeOfAddressChange" need to be changed to "PERMANENT"?  (let me know if I misunderstood and I will resolve this)

Reply:
yes, i lost several changes in the last merge conflict :( thanks!

Reply:
@stiehlrod   Looks like it is still "TEMPORARY",  am I looking for that AC point in the right file, or is it only for the example file in request.json?